content/en/docs/ops/integrations/skywalking/index.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Apache SkyWalking
+description: How to integrate with Apache SkyWalking.
+weight: 32
+keywords: [integration,skywalking,tracing]
+owner: istio/wg-environments-maintainers
+test: no
+---
+
+[Apache SkyWalking](http://skywalking.apache.org) is an application performance monitoring (APM) system, especially designed for
+microservices, cloud native and container-based architectures.
```

Comment:
could you quickly add a sentence why someone would consider using this over the existing observability tools Istio support?